### PR TITLE
bugfix: WEBUMENIA-1194: IE

### PR DESCRIPTION
**Pull request by martin-havala**
vue-sticky added directly, not as npm module because of compilation to ES5

### CODE REVIEW

File: resources/js/sticky.js [hunk 1]
```diff
@@ -0,0 +1,298 @@
+const namespace = '@@vue-sticky-directive';
```
Comment (eronisko) [inline resources/js/sticky.js]: I don't think this is a good idea. The ES6 imports are working for the other packages so I'd bet it should work for this one too.

File: .babelrc [hunk 2]
```diff
@@ -1,3 +1,12 @@
 {
-    "presets": ["@babel/preset-env"]
+    "presets": [
```
Comment (eronisko) [inline .babelrc]: Are you sure this does anything? Isn't this the current setting?

File: resources/views/components/menu.blade.php [hunk 3]
```diff
@@ -58,7 +58,7 @@
 
 @push('scripts')
 <script>
-    $('#menu-button').click(()=> {
+    $('#menu-button').click( function () {
```
Comment (eronisko) [inline resources/views/components/menu.blade.php]: Good catch

**Comment by igor-kamil** [comment]
overall nice. also IE is finally working (again).
just safari and IE doesn't have month aligned to center (maybe just set to parent element text-align: center )

Screenshot 2019-12-13 at 15 21 22

**Pull request merged**